### ISSUE TITLE
Add country Ecuador (EC)

### ISSUE THREAD

**Issue by lauft** [GothenburgBitFactory/holidata#31]
The following locale needs to be verified/completed:
```
class es_EC(Locale):
    """
    01-01: [NF] Año Nuevo
    05-01: [NF] Día del Trabajo
    05-24: [NF] Batalla del Pichincha
    08-10: [NF] Primer Grito de Independencia
    10-09: [NF] Independencia de Guayaquil
    11-02: [NF] Día de Difuntos
    11-03: [NF] Independencia de Cuenca
    12-25: [NRF] Navidad
    : [NV] Carnaval
    : [NRV] Viernes Santo
    """

    locale = "es-EC"
    easter_type = EASTER_WESTERN
```
Checklist:
* Are all holidays (i.e. days "which are defined by law on which business or work are suspended or reduced") listed? Can you provide the legal sources which define the holidays?
* Is the spelling correct? Can you maybe provide a translation?
* Are the fixed dates `<month>-<day>` correct?
* If a holiday has a variable date: Is the algorithm to calculate it correct/can you provide it if missing? If it depends on the easter date: Is it the western or orthodox easter date?
* If a holiday is proclaimed by the government: Can you provide the source?
* Are holidays moved to a different date, e.g. if they fall on a saturday/sunday/...? 
* Are holidays observed on a different date (is there a substitute holiday), e.g. if they fall on a saturday/sunday/...?
* Are the flags correct? (`N` = national holiday, defined by law, `R` = holiday with religious background, `F` = holiday with fixed date, `V` = holiday with variable date)
* Have there been any holidays renamed/deleted/created since 2011? If yes, which ones and since when?

**Comment by alx741** [comment]
> Are all holidays (i.e. days "which are defined by law on which business or work are suspended or reduced") listed? Can you provide the legal sources which define the holidays?

Yes, all correct. *"El ministerio del turismo"*, as the entity responsible for decreeing holidays, lists them in [the official website](https://www.turismo.gob.ec/wp-content/uploads/2020/03/CALENDARIO-DE-FERIADOS.pdf), in compliance with the [public service and labor reformatory law](http://gobiernoabierto.quito.gob.ec/Archivos/Transparencia/2016/12diciembre/A2/ANEXOS/PROCU_LEY_ORGANICA_REFORMATORIA_A_LA_LOSEP_Y_AL_CODIGO_DEL_TRABAJO.pdf) 

> Is the spelling correct? Can you maybe provide a translation?

All correct

> Are the fixed dates <month>-<day> correct?

All correct

> If a holiday has a variable date: Is the algorithm to calculate it correct/can you provide it if missing? If it depends on the easter date: Is it the western or orthodox easter date?

- `[NRV] Viernes Santo`: Orthodox easter date
-  `[NV] Carnaval`:  Weekend that is the closest to new moon between February and March

> If a holiday is proclaimed by the government: Can you provide the source?

Government proclaimed holidays: ["LEY ORGÁNICA REFORMATORIA A LA LEY ORGÁNICA DEL SERVICIOPÚBLICO Y AL CÓDIGO DEL TRABAJO"](http://gobiernoabierto.quito.gob.ec/Archivos/Transparencia/2016/12diciembre/A2/ANEXOS/PROCU_LEY_ORGANICA_REFORMATORIA_A_LA_LOSEP_Y_AL_CODIGO_DEL_TRABAJO.pdf)

> Are holidays moved to a different date, e.g. if they fall on a saturday/sunday/...?

Yes, [Art. 2, Lit. 3 & Art. 3, Lit. 2](http://gobiernoabierto.quito.gob.ec/Archivos/Transparencia/2016/12diciembre/A2/ANEXOS/PROCU_LEY_ORGANICA_REFORMATORIA_A_LA_LOSEP_Y_AL_CODIGO_DEL_TRABAJO.pdf):  `Saturday -> Friday`, `Sunday -> Monday`,  `Tuesday -> Monday`,  `Wednesday/Thursday  -> Friday`.

> Are holidays observed on a different date (is there a substitute holiday), e.g. if they fall on a saturday/sunday/...?

No

> Are the flags correct? (N = national holiday, defined by law, R = holiday with religious background, F = holiday with fixed date, V = holiday with variable date)

All correct

> Have there been any holidays renamed/deleted/created since 2011? If yes, which ones and since when?

No
